### PR TITLE
7787 HOTFIX: Unable to QC document after changing document type from external to internal

### PR DESCRIPTION
**Pull request by rachelschneiderman**

### CODE REVIEW

File: web-client/src/presenter/actions/FileDocument/refreshExternalDocumentTitleFromEventCodeAction.test.js [hunk 1]
```diff
@@ -122,4 +122,52 @@ describe('refreshExternalDocumentTitleFromEventCodeAction', () => {
...
+      {
+        modules: { presenter },
+        state: {
+          form: internalFilingDocument,
+        },
+      },
+    );
+
+    expect(result.state.form.documentTitle).toEqual(
+      internalFilingDocument.documentTitle,
```
Comment (mmarcotte) [inline web-client/src/presenter/actions/FileDocument/refreshExternalDocumentTitleFromEventCodeAction.test.js]: Hmm, I think the runAction might be mutating internalFilingDocument. This test should probably be matching a string and not the object. When I run this locally, they are both getting set to `[Miscellaneous]`

File: web-client/src/presenter/actions/FileDocument/refreshExternalDocumentTitleFromEventCodeAction.test.js [hunk 2]
```diff
@@ -101,25 +89,59 @@ describe('refreshExternalDocumentTitleFromEventCodeAction', () => {
...
+        modules: { presenter },
+        state: {
+          form: {
+            category: 'Miscellaneous',
+            eventCode: 'MISC',
+          },
+        },
+      },
     );
+
```
Comment (mmarcotte) [inline web-client/src/presenter/actions/FileDocument/refreshExternalDocumentTitleFromEventCodeAction.test.js:144]: Hmm these last two tests are the same, but have different names. I think we should test for the secondary document.

**Review by mmarcotte — CHANGES_REQUESTED** [review]

File: web-client/src/presenter/actions/FileDocument/refreshExternalDocumentTitleFromEventCodeAction.test.js [hunk 3]
```diff
@@ -101,25 +89,59 @@ describe('refreshExternalDocumentTitleFromEventCodeAction', () => {
...
+    expect(result.state.form.documentTitle).toEqual('[Miscellaneous]');
   });
```
Comment (mmarcotte) [inline web-client/src/presenter/actions/FileDocument/refreshExternalDocumentTitleFromEventCodeAction.test.js]: These last two tests are the same, but have different names. 

I think we should test for the secondary document.

Comment (rachelschneiderman) [reply web-client/src/presenter/actions/FileDocument/refreshExternalDocumentTitleFromEventCodeAction.test.js]: You are right. I will update that now. Apologies!

**Review by mmarcotte — APPROVED** [review]
Looks great, many thanks!

yipee

**Pull request merged**